examples/passing/PolyLabels.purs

```diff
@@ -0,0 +1,65 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff
+import Control.Monad.Eff.Console
+import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
+
+foreign import unsafeGet
+  :: forall r a
+   . String
+  -> Record r
+  -> a
+
+foreign import unsafeSet
+  :: forall r1 r2 a
+   . String
+  -> a
+  -> Record r1
+  -> Record r2
+
+get
+  :: forall r l a
+   . IsSymbol l
+  => Lookup l r a
+  => SProxy l
+  -> Record r
+  -> a
+get l = unsafeGet (reflectSymbol l)
+
+set
```

Comment:
Actually, this isn't quite right, since `r1` and `r2` are unrelated apart from that they must both contain `l`. We might have to stick to monomorphic updates for now.